test/integration/support/db-helper.js

```diff
@@ -13,6 +13,9 @@ const sampleJob = {
   rating: '1',
   status: 'applied',
   status_sort_index: 1,
+  deadlineDate: null,
+  applicationDate: null,
```

Comment:
as above... Have changed this to be more relevant for the status selected e.g. added status = applied and a applicationDate. Removed the nulls.